File: solutions/1105b.py
import re
from string import ascii_lowercase

k = input().strip().split(' ')[1]
s = input().strip()

maxLevel = 0
for c in ascii_lowercase:
    charLevel = len(re.findall('{0}{{{1}}}'.format(c, k), s))
    maxLevel = max(maxLevel, charLevel)

print(maxLevel)

File: solutions/1105b.pl
$_ = <>;
chomp;
@nk = split/ /;
my $k = @nk[1];
my $s = <>;
chomp $s;
my %levels;
for ('a'..'z') {
	$levels{$_} =()= $s =~ /${_}{${k}}/g;
}
my $max = 0;
for (%levels) {
	$max = $_ if $_ > $max;
}
print $max;
